clients/lsp-php.el

```diff
@@ -352,7 +352,56 @@ already present."
                       (lsp--set-configuration
                        (lsp-configuration-section "serenata"))))
   :server-id 'serenata))
+
+;;; phpactor
 
+(defgroup lsp-phpactor nil
+  "LSP support for Phpactor."
+  :link '(url-link "https://github.com/phpactor/phpactor")
+  :group 'lsp-mode)
+
+(defcustom lsp-phpactor-path "~/.composer/vendor/phpactor/phpactor/bin/phpactor"
+  "Path to the `phpactor' command."
+  :group 'lsp-phpactor
+  :type "string")
+
+(lsp-register-client
+ (make-lsp-client
+  :new-connection (lsp-stdio-connection (list lsp-phpactor-path "language-server"))
```

Comment:
you should change parameter of `lsp-stdio-connection` to lambda instead of static list so user can customize `lsp-phphactor-path` after `lsp-php` has been loaded

Reply:
Fixed.